compiler/src/dotty/tools/dotc/tastyreflect/SymbolOpsImpl.scala

```diff
@@ -1,14 +1,49 @@
-package dotty.tools.dotc.tastyreflect
+package dotty.tools.dotc
+package tastyreflect
 
 import dotty.tools.dotc.core.Symbols._
 
 trait SymbolOpsImpl extends scala.tasty.reflect.SymbolOps with TastyCoreImpl {
 
   def SymbolDeco(symbol: Symbol): SymbolAPI = new SymbolAPI {
-    def isEmpty: Boolean = symbol eq NoSymbol
-    def localContext(implicit ctx: Context): Context = ctx.withOwner(symbol)
+
+    def isEmpty(implicit ctx: Context): Boolean = symbol eq NoSymbol
```

Comment:
Can we rename it to `exists`? Not important, somehow I feel `exists` is more friendly.

Reply:
Renamed